src/lug/Window/main.cpp

```diff
@@ -0,0 +1,12 @@
+#include <lug/Window/Window.hpp>
```

Comment:
Remove this useless file (use a new sample if you want)
And please, make a decent example, not one with a `while (1)` :)